diy-omd.ino

```diff
@@ -85,6 +106,244 @@ void loop() {
     Serial.println("Temp C @ Index 0: " + String(temperatureSensors.getTempCByIndex(0))); // Get the first temperature.
   }
 
+  processIncomingBTData();
+  operateMotor();
+  processHallSensor();
+  logRPM();
+
+  if (testMillis > 500) {
+    test++;
+    transmitBTData("temp", (String)test);
+    testMillis = 0;
+  }
```

Comment:
Maybe we move the time check into the method like the others right above, so the call here is just to a function that sends some bits of data via Bluetooth